tests/integration/test_main.py

```diff
@@ -143,12 +177,23 @@ def test_generate_measures_dummy_tables(tmp_path):
         environ={},
         user_args=(),
     )
-    assert output_file.read_text() == textwrap.dedent(
-        """\
-        measure,interval_start,interval_end,ratio,numerator,denominator,sex
-        births,2020-01-01,2020-12-31,1.0,1,1,male
-        births,2020-01-01,2020-12-31,0.0,0,1,female
-        births,2021-01-01,2021-12-31,0.0,0,1,male
-        births,2021-01-01,2021-12-31,1.0,1,1,female
-        """
-    )
+    if disclosure_control_enabled:
+        assert output_file.read_text() == textwrap.dedent(
+            """\
+            measure,interval_start,interval_end,ratio,numerator,denominator,sex
+            births,2020-01-01,2020-12-31,,0,0,male
+            births,2020-01-01,2020-12-31,,0,0,female
+            births,2021-01-01,2021-12-31,,0,0,male
+            births,2021-01-01,2021-12-31,,0,0,female
+            """
+        )
+    else:
+        assert output_file.read_text() == textwrap.dedent(
+            """\
+            measure,interval_start,interval_end,ratio,numerator,denominator,sex
+            births,2020-01-01,2020-12-31,1.0,1,1,male
+            births,2020-01-01,2020-12-31,0.0,0,1,female
+            births,2021-01-01,2021-12-31,0.0,0,1,male
+            births,2021-01-01,2021-12-31,1.0,1,1,female
+            """
+        )
```

Comment:
As these multiline strings are re-used, would it make sense to have them as module-level constants?

Reply:
They're not, now, because `test_generate_measures` has richer data.